tasks/Debian-tarball.yml

```diff
@@ -0,0 +1,12 @@
+---
+- include: tarball.yml
+
+- include: upstart.yml
+  tags: deploy
+
+- include: systemd.yml
+  tags: deploy
```

Comment:
Note: I didn't test the deployment with a later Ubuntu (>15.10 or 16.04 where it's default) which actually runs systemd  (though the systemd template file  looks quite basic, so I'ld expect it to work)